stories/Tile.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import { Tile } from "../src/components/Tile/Tile";
+
+storiesOf("Tile", module).add("tile props", () => {
+  const props = {
+    link: "www.hackney.gov.uk",
+    title: "Mrs Joan Fisher",
+    details: [
+      { DateOfBirth: "1970 - 02 - 30" },
+      { Mobile: "07777123456" },
+      { Home: "02088881234" },
+      { Work: "02012345678" },
+    ],
+    email: "mjf@email.com",
+  };
+
+  return (
+    <Tile
+      link={props.link}
+      title={props.title}
+      details={props.details}
```

Comment:
The type error that's coming off this would be resolved if you used props.children instead 👍